INSTALL-LINUX.md

```diff
@@ -1,66 +1,74 @@
 ## Prerequisites
 
-You'll need the following requirements:
-
-- Python 3.6+
-- Git (`git` package on Ubuntu distributions)
-- Python libraries: `venv` (`python3-venv` package on Ubuntu distributions)
-
-You can get these on a Debian based distribution using `apt-get`
-
+To use and/or contribute to PyBaMM, you must have Python 3.6 or above installed.
+To install Python 3 on Debian-based distribution (Debian, Ubuntu, Linux mint), open a terminal and run
+```bash
+sudo apt update
+sudo apt install python3
+```
+On Fedora or CentOS, you can use DNF or Yum. For example
 ```bash
-sudo apt-get install python3 git-core python3-venv
+sudo dnf install python3
 ```
 
 ## Install PyBaMM
 
-The first step is to get the code by cloning this repository
-
+## User install
+PyBaMM can be installed via pip:
 ```bash
-git clone https://github.com/pybamm-team/PyBaMM.git
-cd PyBaMM
+pip install pybamm
 ```
 
-The safest way to install PyBaMM is to do so within a virtual environment ([introduction
-to virtual environments](https://realpython.com/python-virtual-environments-a-primer/)).
+We recommend to install PyBaMM within a virtual environment, in order not
+to alter any distribution python files.
 To create a virtual environment `env` within your current directory type:
 
 ```bash
 python3 -m venv env
 ```
-
 You can then "activate" the environment using:
 
 ```bash
 source env/bin/activate
 ```
-
 Now all the calls to pip described below will install PyBaMM and its dependencies into
 the environment `env`. When you are ready to exit the environment and go back to your
 original system, just type:
 
 ```bash
 deactivate
 ```
```

Comment:
I would put this before the "user install" section as otherwise someone might do `pip install pybamm` before setting up a venv (and it also applies to developer install)